packages/ui-pattern-block/src/components/JsonErrorMessage.tsx

```diff
@@ -0,0 +1,16 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { IconExclamationMarkTriangle20 } from '@frontify/fondue';
+import { ReactElement } from 'react';
+
+export const JsonErrorMessage = (): ReactElement => {
+    return (
+        <div
+            data-test-id="dependency-invalid-json"
+            className="tw-text-box-negative-inverse tw-w-full tw-h-11 tw-flex tw-items-center tw-px-3 tw-gap-3 tw-bg-box-negative tw-text-s tw-font-medium tw-font-sans"
+        >
+            <IconExclamationMarkTriangle20 />
+            <span>The entered JSON is invalid</span>
```

Comment:
I think we need to announce that error message to screen readers for accessibility, probably with a `aria-live="assertive"`, would be nice if you could ask @LidijaStoilova about that

Reply:
Thanks ,added